src/add_task/add_task.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+export default function AddTask(props) {
+    let taskInput = React.createRef();
```

Comment:
Interesting - what are the tradeoffs here between using a `ref` or, alternatively, state with an `onChange` event? Which style is most likely to be seen as idiomatic?